app/server-utils/stampy.ts

```diff
@@ -115,3 +137,55 @@ export const loadAllCanonicallyAnsweredQuestions = withCache(
     return data
   }
 )
+
+export const loadAllQuestions = withCache(
+  'allQuestions',
+  async () => {
+    const rows = await getCodaRows('All answers')
+    return rows.map(({ name }) => name)
+  }
+)
+
+
+export const insertRows = async (
+  table: string,
+  rows: NewQuestion[],
+) => {
+  const url = `https://coda.io/apis/v1/docs/${CODA_DOC_ID}/tables/${enc(table)}/rows`
+  const payload = {
+    'rows': rows.map(row => (
+        {
+          'cells': [
+            { 'column': 'c-Z1ouuaH5zW', 'value': row.title },
```

Comment:
would these be better as constants? Or a helper function?

Reply:
is it possible to insert rows via API by column name instead of column id?

if not, we should probably have the ids in constants as you suggest,, but not just for writing, also for the GET request (to use the same mechanism and not 2 unsynchronized ways to map columns)

Reply:
turns out that you can use column names. As long as no one changes them. Should be fine